docs/core/testing/unit-testing-mstest-sdk.md

```diff
@@ -140,6 +140,54 @@ Or to disable an extension that is coming from the selected profile. In this cas
 </Project>
 ```
 
+## Features
+
+Outside of the selection of the runner and runner specific extensions, `MSTest.Sdk` also provides additional features to simplify and enhance your testing experience.
+
+### Aspire
+
+`.NET Aspire` is an opinionated, cloud ready stack for building observable, production ready, distributed applications. .NET Aspire is delivered through a collection of NuGet packages that handle specific cloud-native concerns. For more information about it, check out its [documentation page](https://learn.microsoft.com/dotnet/aspire/get-started/aspire-overview).
```

Comment:
```suggestion
.NET Aspire is an opinionated, cloud ready stack for building observable, production ready, distributed applications. .NET Aspire is delivered through a collection of NuGet packages that handle specific cloud-native concerns. For more information, see the [.NET Aspire docs](/dotnet/aspire/get-started/aspire-overview).
```

Reply:
Oooooh! I was looking on disk and I couldn't find this path that's why I used the full URL.